lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -789,7 +788,15 @@ def quickchip_payment?(payment)
 
       def card_from_response(response)
         # StripePI puts the AVS and CVC check significantly deeper into the response object
-        response['card'] || response['active_card'] || response['source'] || response.dig('charges', 'data', 0, 'payment_method_details', 'card', 'checks') || {}
+        response['card'] || response['active_card'] || response['source'] ||
+          response.dig('charges', 'data', 0, 'payment_method_details', 'card', 'checks') ||
+          response.dig('latest_attempt', 'payment_method_details', 'card', 'checks') {}
```

Comment:
Stripe nests this data in 5 spots. Insanity 🤯

Reply:
some of those spots may be historical, but still.

Reply:
yeah it is wild.

Reply:
Geez Louise